### PR TITLE
INT-6550 introduce minimum required resources

### PR DESCRIPTION
**Pull request by jflinchbaugh**
provide minimum requirements in docs and as samples in values.yaml to make it easier for users to setup for a production instance.

JIRA: https://issues.sonatype.org/browse/INT-6550
Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/Helm3%20Charts/job/Feature%20Snapshot%20Builds/job/INT-6550-prod-resources/

### CODE REVIEW

File: charts/nexus-iq/values.yaml [hunk 1]
```diff
@@ -28,21 +28,6 @@ iq:
...
-    timeoutSeconds: 2
-    successThreshold: 1
-  # Configures the readiness probe for IQ pod
-  readinessProbe:
-    initialDelaySeconds: 30
-    periodSeconds: 30
-    failureThreshold: 10
-    timeoutSeconds: 2
-    successThreshold: 1
-
```
Comment (jflinchbaugh) [inline charts/nexus-iq/values.yaml:45]: minor reordering for clarity.

File: charts/nexus-iq/README.md [hunk 2]
```diff
@@ -69,41 +69,42 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `persistence.accessMode`                   | Default access mode                                                                                                 | `ReadWriteOnce`                                             |
+| `persistence.existingClaim`                | Pre-created PVC name for Data Volume                                                                                | `nil`                                                       |
+| `persistence.existingLogClaim`             | Pre-created PVC name for Log Volume                                                                                 | `nil`                                                       |
+| `persistence.volumeConfiguration`          | A YAML block to configure the persistent volume type. Defaults to `hostPath` which should not be used in production | `hostPath`                                                  |
+| `resources'`                               | Resource requests and limits for the IQ pod in the cluster.                                                         | See `values.yaml` for suggested minimum recommended values. |
```
Comment (jflinchbaugh) [inline charts/nexus-iq/README.md:107]: editor lined everything up.  this last line is the only real addition.

File: charts/nexus-iq/README.md [hunk 3]
```diff
@@ -69,41 +69,42 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `persistence.accessMode`                   | Default access mode                                                                                                 | `ReadWriteOnce`                                             |
+| `persistence.existingClaim`                | Pre-created PVC name for Data Volume                                                                                | `nil`                                                       |
+| `persistence.existingLogClaim`             | Pre-created PVC name for Log Volume                                                                                 | `nil`                                                       |
+| `persistence.volumeConfiguration`          | A YAML block to configure the persistent volume type. Defaults to `hostPath` which should not be used in production | `hostPath`                                                  |
+| `resources`                                | Resource requests and limits for the IQ pod in the cluster.                                                         | See `values.yaml` for suggested minimum recommended values. |
```
Comment (jflinchbaugh) [inline charts/nexus-iq/README.md:107]: editor lined everything up. this last line is the only real addition.

**Review by eduard-tita — APPROVED** [review]
+1  LGTM

**Review by hectorhuol — APPROVED** [review]
+1 LGTM

**Pull request merged**